src/secure_random.cr

```diff
@@ -1,19 +1,66 @@
 require "base64"
 require "openssl/lib_crypto"
 
+# The SecureRandom library is an interface for creating secure random values in different formats, 
+# including Base64 string, hexadecimal string, uuid and others. It is implemented with `OpenSSL` as well as 
+# `LibCrypto` libraries.
```

Comment:
libcrypto usually comes as part of OpenSSL. I was more thinking of something like "libcrypto (OpenSSL) is used as the source of the random numbers."